autowebcompat/utils.py

```diff
@@ -179,12 +179,17 @@ def make_infinite(gen_func, elems):
         yield from gen_func(elems)
 
 
-def read_labels(file_name='labels.csv'):
+def read_labels(file_name='labels.csv', classification_type = None):
```

Comment:
I prefer a separate function to handle this, rather than overloading read_labels with additional meanings.

Reply:
@marco-c  what if instead of writing a seperate function we directly modified content using 

``` python
            if classification_type == 'Y vs D + N':
                labels = {x: 1 if y == 'y' else 0 for x,y in labels.items}
            elif classification_type == 'Y + D vs N':
                labels = {x: 0 if y == 'n' else 1 for x,y in labels.items}
```

or I could pass the dictionary to a simple function and overwrite the labels, instead of making a new dictionary and returning it like I did earlier.

Reply:
> @marco-c what if instead of writing a seperate function we directly modified content using

Better as a function, so you can easily write some test for it.

> I could pass the dictionary to a simple function and overwrite the labels, instead of making a new dictionary and returning it like I did earlier.

It seems simpler to me to just have a function that, given a label, returns the categorical value.